test/appframework/aws/m4/appframework_aws_suite_test.go

```diff
@@ -50,7 +50,7 @@ var (
 	currDir, _            = os.Getwd()
 	downloadDirV1         = filepath.Join(currDir, "m4appfwV1-"+testenv.RandomDNSName(4))
 	downloadDirV2         = filepath.Join(currDir, "m4appfwV2-"+testenv.RandomDNSName(4))
-	downloadDirPVTestApps = filepath.Join(currDir, "m4appfwPVTestApps-"+testenv.RandomDNSName(4))
+	downloadDirPVTestApps = filepath.Join(currDir, "c3appfwPVTestApps-"+testenv.RandomDNSName(4))
```

Comment:
NIT: should the names not be changed. This is the m4 directory

Reply:
Indeed, bad copy-paste